pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/ValidDocIdMetadataInfo.java

```diff
@@ -28,14 +28,19 @@ public class ValidDocIdMetadataInfo {
   private final long _totalValidDocs;
   private final long _totalInvalidDocs;
   private final long _totalDocs;
+  private final String _segmentCrc;
+  private final String _validDocIdType;
```

Comment:
Rename to `_validDocIdsType`?

Also suggest introducing an enum

Reply:
renamed everything to `validDocIdsType`